frontend/src/views/pages/dealerships/Dealership.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div>
+    <CCol>
+      <CCard>
+        <CCardBody>
+          <div v-if="!editingDealership">
+            <h1>{{ dealership.name }}</h1>
+            <h2 v-if="dealership.admin">
+              {{ dealership.admin.first_name }}'s dealership
+            </h2>
+            <p>{{ dealership.description }}</p>
+            <router-link to="/dealerships">
+              <CButton color="primary"> Back to list </CButton>
+            </router-link>
+            <br />
+            <CButton
+              id="edit-dealership"
+              v-if="!editingDealership"
+              color="secondary"
+              class="mt-2"
+              @click="setEditingDealership(true)"
+              >Edit dealership details</CButton
+            >
+          </div>
+          <edit-dealership
+            v-if="editingDealership"
+            :dealership="dealership"
+            :setEditingDealership="setEditingDealership"
+          />
+        </CCardBody>
+        <CCardFooter>
+          <small class="text-muted">
+            {{ dealership.created_at }}
+          </small>
+        </CCardFooter>
+      </CCard>
+      <CRow>
+        <div class="col-md-4">
+          <CAlert v-if="!$store.state.auth.createUserCompleted" color="success"
+            >Begin by creating your staff by clicking the
+            <strong>"Create a staff account"</strong> button below.</CAlert
+          >
+        </div>
+      </CRow>
+      <dealership-accounts v-if="userHasPermissions('View Staff Users')" />
+      <dealership-vehicle-properties
+        v-if="userHasPermissions('View Vehicle Properties')"
+      />
+    </CCol>
+  </div>
+</template>
+
+<script>
+import EditDealership from "./EditDealership.vue";
+import DealershipAccounts from "../../../modules/users/DealershipAccounts.vue";
+import DealershipVehicleProperties from "../../../modules/vehicleProperties/DealershipVehicleProperties.vue";
+
+const axios = require("axios");
+const { containsPermissions } = require("../../../utils/index");
+
+export default {
+  name: "Dealership",
+  data() {
+    return {
+      dealership: {},
+      editingDealership: false,
+    };
+  },
+  methods: {
+    setEditingDealership(value) {
+      this.editingDealership = value;
+    },
+    userHasPermissions(...permissions) {
+      // console.log(permissions);
+      // console.log(containsPermissions(permissions));
+      return containsPermissions(permissions);
```

Comment:
Removed commented out code

Reply:
Fixed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/37/commits/8c83bea65e9c8a6186a069f6415340392a55a7b4)